src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -16,6 +16,7 @@
 
 package org.terasology.launcher.gui.javafx;
 
+import com.sun.javafx.scene.control.skin.ComboBoxListViewSkin;
```

Comment:
This import is not nice, but the only workaround we've found 🤷‍♂ works with the bundled Liberica JDK.